doc/_static/style.css

```diff
@@ -7,12 +7,32 @@ body {
 }
 
 a {
-    color: rgb(255,2,16);
+    color: #08C;
     text-decoration: none;
 }
 
 a:hover {
-    color: rgb(255,2,16);
+    color: rgb(102,178,255);
+}
+
+h2 a {
+    color: rgb(0,0,0);
+    text-decoration: none;
+}
+
+h1 {
+    font-weight: 1000;
+}
+
+h2 {
+    background-color: rgb(255,255,255);
+    font-size: 140%;
+    font-weight: 600;
+}
+
+h3 {
+    font-size: 110%;
+    font-weight: 150;
```

Comment:
Can you practically mention what this changes? Does it make things more legible, change the color scheme, etc.?